mobile/src/components/Report.js

```diff
@@ -106,8 +106,23 @@ export default class Report extends Component {
     apiClient.get(url)
       .then((response) => {
         if (response) {
+          const { colModel, totalPages } = response[0];
+          const templateParameters = [];
+          Object.keys(colModel[0]).forEach((colName) => {
+            const { order } = colModel[0][colName][0];
+            const parameter = this.state.templateParameters.find(param =>
+              param.name === colName);
+            if (parameter) {
+              templateParameters.push({
+                ...parameter,
+                order,
```

Comment:
you should parse the order here, not during sorting

Reply:
ok